tests/namespaced-admission-policy-tests.bats

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env bats
 
-source $BATS_TEST_DIRNAME/common.bash
+setup() {
+	load common.bash
+	wait_pods
+}
 
-setup_file() {
-	kubectl --context $CLUSTER_CONTEXT delete --wait --ignore-not-found pods --all
-	kubectl --context $CLUSTER_CONTEXT delete --wait --ignore-not-found -n kubewarden clusteradmissionpolicies --all
-	kubectl --context $CLUSTER_CONTEXT delete --wait --ignore-not-found -n kubewarden admissionpolicies --all
-	kubectl --context $CLUSTER_CONTEXT wait --for=condition=Ready -n kubewarden pod --all
+teardown_file() {
+	kubectl delete --wait --ignore-not-found pods --all
```

Comment:
same as https://github.com/kubewarden/kubewarden-end-to-end-tests/pull/32/files#r934599224